crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -71,6 +72,7 @@ pub struct AuthorityPerpetualTables {
     /// to be executed, we wait for them to appear in this table. When we revert transactions, we remove them from both
     /// tables.
     pub(crate) executed_effects: DBMap<TransactionDigest, TransactionEffectsDigest>,
+    pub(crate) events: DBMap<TransactionEventsDigest, TransactionEvents>,
```

Comment:
hmm I was under the impression that we are not going to store events on validators

Reply:
if this is not stored, `handle_certificate` might not return the events if the cert has been executed already, is that ok?

Reply:
Good Q. I kind of forgot what was our conclusion on this. At least we can store it for now and figure out what to do latter. Can you add a TODO to this table? We will also need to figure out the pruning policy for it.